echo/server/dembrane/report_utils.py

```
@@ -1,14 +1,28 @@
+import re
 import logging
 
+from litellm import completion, token_counter
 from sqlalchemy.orm import Session
```

Ruff (0.8.2)

`sqlalchemy.orm.Session` imported but unused

Remove unused import: `sqlalchemy.orm.Session`

(F401)